src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -204,7 +204,6 @@ public final class InlineConfigParser {
      *  Until <a href="https://github.com/checkstyle/checkstyle/issues/15456">#15456</a>.
      */
     private static final Set<String> SUPPRESSED_CHECKS = Set.of(
-            "com.puppycrawl.tools.checkstyle.checks.annotation.PackageAnnotationCheck",
```

Comment:
This is unrelated to the current issue (remove //ok)

Reply:
yes, please move it to separate PR, I will mere this quickly

Reply:
oh I see , it is for src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/annotation/packageannotation/InputPackageAnnotation2.java

Reply:
you mixing few upadate in one commit, not good. please separate